e2e/cypress/integration/keyboard_shortcuts/ctrl_cmd_k_open_channel_from_global_threads_spec.js

```diff
@@ -0,0 +1,119 @@
+            testChannel = channel;
+
+            cy.apiSaveCRTPreference(testUser.id, 'on');
+            cy.apiCreateUser({prefix: 'other'}).then(({user: user1}) => {
+                otherUser = user1;
+
+                cy.apiAddUserToTeam(testTeam.id, otherUser.id).then(() => {
+                    cy.apiAddUserToChannel(testChannel.id, otherUser.id);
+                });
+            });
+        });
+    });
+
+    beforeEach(() => {
+        // # Visit channel
+        cy.visit(`/${testTeam.name}/channels/${testChannel.name}`);
+    });
+
+    it('MM-T1243 CTRL/CMD+K - Create thread, Open global threads and then from find channels switch channel using arrow keys and Enter', () => {
+        // # Post message as other user
+        cy.postMessageAs({
+            sender: otherUser,
+            message: 'Root post,',
+            channelId: testChannel.id,
+        }).then(({id: rootId}) => {
+            // * Thread footer should not be visible
+            cy.get(`#post_${rootId}`).find('.ThreadFooter').should('not.exist');
+
+            // # Click on post to open RHS
+            cy.get(`#post_${rootId}`).click();
+
+            // * Button on header should say Follow as current user is not following
+            cy.get('#rhsContainer').find('.FollowButton').should('have.text', 'Follow');
+
+            // # Post a reply as current user
+            cy.postMessageReplyInRHS('Reply!');
+
+            // # Get last root post
+            cy.get(`#post_${rootId}`).
+
+                // * thread footer should exist now
+                get('.ThreadFooter').should('exist').
+                within(() => {
+                // * the button on the footer should say Following
+                    cy.get('.FollowButton').should('have.text', 'Following');
+                });
+
+            // * the button on the RHS header should now say Following
+            cy.get('#rhsContainer').find('.FollowButton').should('have.text', 'Following');
+
+            // # Visit global threads
+            cy.uiClickSidebarItem('threads');
+
+            // * There should be a thread there
+            cy.get('article.ThreadItem').should('have.have.lengthOf', 1);
+        });
+
+        // # Press CTRL/CMD+K
+        cy.get('body').cmdOrCtrlShortcut('K');
+        cy.get('#quickSwitchInput').type('T');
+
+        // # Press down arrow
+        cy.wait(TIMEOUTS.HALF_SEC);
+        cy.get('body').type('{downarrow}');
+
+        // * Confirm the offtopic channel is selected in the suggestion list
+        cy.get('#suggestionList').findByTestId('off-topic').should('be.visible').and('have.class', 'suggestion--selected');
+
+        // # Press up arrow
+        cy.wait(TIMEOUTS.HALF_SEC);
+        cy.get('body').type('{uparrow}');
+
+        // * Confirm the townsquare channel is selected in the suggestion list
+        cy.get('#suggestionList').findByTestId('town-square').should('be.visible').and('have.class', 'suggestion--selected');
```

Comment:
Can you also assert that the URL doesn't change?
i.e: The URL would change if threads list consumed the "up/down arrow" event.
We'll need 2 threads for that.